src/search-modal/data/apiHooks.js

```diff
@@ -45,6 +45,9 @@ export const useContentSearchResults = ({
   blockTypesFilter,
   tagsFilter,
 }) => {
+  blockTypesFilter ??= []; // eslint-disable-line no-param-reassign -- default value for optional parameter
+  tagsFilter ??= []; // eslint-disable-line no-param-reassign -- Default value for optional parameter
+
```

Comment:
There's a simpler way to write this:

```suggestion
  blockTypesFilter = [],
  tagsFilter = [],
}) => {
```

Reply:
Thank you @bradenmacdonald!
Done: 7d6096e2cea78aa79497305058d019fda31e5a35